test/backend_test.in.cpp

```diff
@@ -51,6 +51,33 @@ static const vector<element::Type> s_known_element_types = {element::from<float>
                                                             element::from<uint32_t>(),
                                                             element::from<uint64_t>()};
 
+class UnhandledOp : public ngraph::op::Op
+{
+public:
+    UnhandledOp(const std::shared_ptr<Node>& arg)
+        : Op("Unsupported_op", {})
+    {
+        set_value_type_checked(arg->get_element_type(), arg->get_shape());
```

Comment:
`set_value_type_checked` has been removed from `ngraph::Node`, see the new API for adding outputs to ops.